### ISSUE TITLE
[Feat]: add new endpoint to replace `GET /session/:sessionId`

### ISSUE THREAD

**Issue by eglitise** [appium/appium#19858]
Based on #19851. These are just ideas and the exact endpoint values are open to discussion.

`GET /session/:sessionId` => `GET /session/:sessionId/appium/info`
~~`GET /session/:sessionId/log` => `GET /session/:sessionId/appium/log`~~ (already exists in Selenium as `/se/log`)
~~`GET /session/:sessionId/log/types` => `GET /session/:sessionId/appium/appium/types`~~ (already exists in Selenium as `/se/log/types`)

**Comment by KazuCocoa** [comment]
for logs, we already re-use `/se` endpoints to keep client's selenium clients compatibility

**Comment by eglitise** [comment]
Good point, I was not aware of that. Then I guess there's only the session info endpoint left?

**Comment by KazuCocoa** [comment]
Yes, i think so.

Btw, I saw the `/session/<session id>` endpoint was proposed once in the past in the w3c in https://github.com/w3c/webdriver
I don't recall right now about the details but the propose was closed after some discussion.

**Comment by eglitise** [comment]
Moving the discussion to #20880
